### PR TITLE
udpate: several fixes/improvements to German translation

### PR DESCRIPTION
**Pull request by socialmedialabs**
This PR/Translation is about #ISSUE_NUMBER.

**files translated**:
  - [x] `def/--LOCALE-CODE--/about.cson`
  - [x] `def/--LOCALE-CODE--/context.cson`
  - [x] `def/--LOCALE-CODE--/menu_darwin.cson`
  - [x] `def/--LOCALE-CODE--/menu_linux.cson`
  - [x] `def/--LOCALE-CODE--/menu_win32.cson`
  - [x] `def/--LOCALE-CODE--/settings.cson`
  - [x] `def/--LOCALE-CODE--/welcome.cson`

**validation**
  - [x] I've validated my translation locally by running

`npm run validation -- --locale MY_LOCALE`

### CODE REVIEW

**Comment by socialmedialabs** [comment]
I have changed translated terms where I felt they were kinda wonky. Please feel free to merge. Planning to add/improve translation further on for German.

File: def/de/settings.cson [hunk 1]
```diff
@@ -138,51 +138,36 @@ Settings:
...
-        desc: "Dem Senden und der Verwendung von statistischen Daten sowie Ausnahmemeldungen an das Atom Team zustimmen. Dient der Produkt um mitzuhelfen das Produkt zu verbessern."
+        title: "Sende Telemetrie-Daten an das Atom-Team"
+        desc: "Dem Senden und der Verwendung von statistischen Daten sowie Ausnahmemeldungen an das Atom-Team zustimmen. Dies hilft uns, um Atom noch besser zu machen."
         select:
           limited: "Limitierte anonyme Nutzer-Statistiken, Ausnahmen und Fehlerberichte"
           no: "Keine Telemetrie-Daten senden"
-          undecided: "Unentschlossen (Atom fragt beim nächsten Start erneut nach)"
-      }
-      {
-        _id: 'core.titleBar'
```
Comment (liuderchi) [inline def/de/settings.cson]: Please keep this section `core.titleBar` to keep consistency of `settings.cson` of all locales.
this part may appear in older versions.

File: def/de/settings.cson [hunk 2]
```diff
@@ -138,51 +138,36 @@ Settings:
...
-          hidden: "hidden"
+          undecided: "Weiß nicht (Atom fragt beim nächsten Start erneut nach)"
       }
       {
         _id: 'core.useProxySettingsWhenCallingApm'
         title: "Nutze die Proxy-Einstellungen, wenn APM aufgerufen wird"
-        desc: "Nutze die erkannten Proxy-Einstellungen, wenn das <code>apm</code> Kommandozeilenwerkzeug aufgerufen wird."
-      }
-      {
-        _id: 'core.versionPinnedPackages'
```
Comment (liuderchi) [inline def/de/settings.cson]: Please keep this section `core. versionPinnedPackages ` to keep consistency of `settings.cson` of all locales.
this part may appear in older versions.

**Comment by socialmedialabs** [comment]
My bad, this is fixed now. I re-introduced the missing sections. Thanks!


Von: Te-Chi Liu <notifications@github.com>
Antworten: liuderchi/atom-i18n <reply@reply.github.com>
Datum: 7. Oktober 2017 at 18:01:17
An: liuderchi/atom-i18n <atom-i18n@noreply.github.com>
CC: social media labs <info@socialmedialabs.de>, Author <author@noreply.github.com>
Betreff:  Re: [liuderchi/atom-i18n] udpate: several fixes/improvements to German translation (#109)  

@liuderchi requested changes on this pull request.

Please keep two sections in settings.cson to keep consistency of all locales. These parts may appear in older versions.

In def/de/settings.cson:

>          select:
           limited: "Limitierte anonyme Nutzer-Statistiken, Ausnahmen und Fehlerberichte"
           no: "Keine Telemetrie-Daten senden"
-          undecided: "Unentschlossen (Atom fragt beim nächsten Start erneut nach)"
-      }
-      {
-        _id: 'core.titleBar'
Please keep this section core.titleBar to keep consistency of settings.cson of all locales.
this part may appear in older versions.

In def/de/settings.cson:

>        }
       {
         _id: 'core.useProxySettingsWhenCallingApm'
         title: "Nutze die Proxy-Einstellungen, wenn APM aufgerufen wird"
-        desc: "Nutze die erkannten Proxy-Einstellungen, wenn das <code>apm</code> Kommandozeilenwerkzeug aufgerufen wird."
-      }
-      {
-        _id: 'core.versionPinnedPackages'
Please keep this section core. versionPinnedPackages to keep consistency of settings.cson of all locales.
this part may appear in older versions.

**Comment by socialmedialabs** [comment]
I'd need see them in context. Otherwise it's almost impossible to translate them to something meaningful.

I don't want to translate the sections in literal as they would become more confusing than the original text.

Any suggestions?

**Comment by liuderchi** [comment]
@socialmedialabs thanks for the correction. 🎉 
would you like to help translate those added sections?

**Comment by socialmedialabs** [comment]
Thanks. I will keep adding translations once I complete another set of sections.

**Pull request merged**